crates/swc_html_parser/src/parser/mod.rs

```diff
@@ -1828,6 +1855,7 @@ where
                         self.input.set_input_state(State::ScriptData);
                         self.original_insertion_mode = self.insertion_mode.clone();
                         self.insertion_mode = InsertionMode::Text;
+                        maybe_allow_self_closing!(is_self_closing, tag_name);
```

Comment:
I think we don't need it here, because it is the `script` tag, I don't think it is possible to have `<script />`

Reply:
I was generally thinking the same before I recalled this usage: https://vuejs.org/api/sfc-spec.html#src-imports.

TBH, I'd rather agree with you and make `<script />` and `<style />` unsupported. I've rarely seen src imports ever being used.